CONTRIBUTING.md

```diff
@@ -47,8 +47,8 @@ Rake can be found at http://docs.seattlerb.org/rake/.
 
 To run a single test suite
 
-   rake test TEST=path/to/test.rb
+`rake test TEST=path/to/test.rb`
```

Comment:
On the other hand @bf4 $ make explicit the ideia that is bash command instead of ruby, for instance.